src/mechs/hkdf/hkdf.ts

```diff
@@ -0,0 +1,71 @@
+import * as core from "webcrypto-core";
+import { HmacCryptoKey } from "../hmac/key";
+import { HkdfCryptoKey } from "./key";
+import { BufferSourceConverter, CryptoKey } from "webcrypto-core";
+import crypto from "crypto";
+
+export class HkdfProvider extends core.HkdfProvider {
+
+  private normalizeHash(hash: HashAlgorithmIdentifier): Algorithm {
+    if (typeof hash === "string") {
+      hash = {name: hash};
+    }
+
+    this.checkHashAlgorithm(hash, this.hashAlgorithms);
+    return hash;
+  }
+
+  public async onImportKey(format: KeyFormat, keyData: ArrayBuffer, algorithm: HmacImportParams, extractable: boolean, keyUsages: KeyUsage[]): Promise<CryptoKey> {
+    if (format.toLowerCase() !== "raw") {
+        throw new core.OperationError("Operation not supported");
+    }
+
+    const key: HkdfCryptoKey = new HkdfCryptoKey();
+    key.data = Buffer.from(keyData);
+    key.algorithm = { name: this.name };
+    key.extractable = extractable;
+    key.usages = keyUsages;
+    return key;
+  }
+
+  public async onExportKey(format: KeyFormat, key: HmacCryptoKey): Promise<JsonWebKey | ArrayBuffer> {
```

Comment:
given that W3C forces `extractable` to be `false` during import, is this worthy to implement?

Reply:
`webcrypto-core` doesn't allow to create extractable HKDF key. I think we can remove `onExportKey` method